src/main.cpp

```diff
@@ -990,17 +1002,10 @@ int main ( int argc, char** argv )
     catch ( const CGenErr& generr )
     {
         // show generic error
-#ifndef HEADLESS
-        if ( bUseGUI )
-        {
-            QMessageBox::critical ( nullptr, APP_NAME, generr.GetErrorText(), "Quit", nullptr );
```

Comment:
This is the only place in the code where the messagebox button originally doesn't have a "Ok" text, though the original "Quit" would expect the program to terminate when clicking it, which it doesn't.

If we really want to 'Quit' here it should be a modal messagebox and we should really have to exit on pressing 'Quit'

Reply:
Is CGenErr critical as in Jamulus would be unusable if this error occurs? Probably not if we didn't quit the program before. Do you consider this a bug you fixed?

Reply:
> Is CGenErr critical as in Jamulus would be unusable if this error occurs? 

CGenErr is only to retrieve the error message from an exception...

> Probably not if we didn't quit the program before. Do you consider this a bug you fixed?

No I didn't fix anything here. I just note that the button text here is different, but the original button text is misleading.

Reply:
Thanks for clarifying. 

Yes you did fix a linguistic "bug".